saml-authentication-server/src/main/java/jetbrains/buildServer/auth/saml/plugin/SamlAuthenticationScheme.java

```diff
@@ -139,6 +144,22 @@ public HttpAuthenticationResult processAuthenticationRequest(@NotNull HttpServle
 
                             if (user == null) {
                                 LOG.warn(String.format("New user %s was not created due to unknown reason", username));
+                            } else {
```

Comment:
Please note, that placing the logic under this conditional branch only enables it for newly created users. It won't allow changing groups set for existing users.

Reply:
I've moved this logic as part of [`5a2d82e` (#52)](https://github.com/morincer/teamcity-plugin-saml/pull/52/commits/5a2d82e3e9167f316bb5d31cf2a24c11c706b4dc) in order to put it behind a boolean setting. 

Will need to add logic to remove groups aswell..